### ISSUE TITLE
Problem with background definition in the qml presentation component

### ISSUE THREAD

**Issue by hiukky** [calamares/calamares#1649]
**Describe the bug**
I'm creating my branding for squids, where the theme is totally dark! I was successful in styling other components, but in the Slide component I was unable to reach the result!

Any change to "show.qml" that involves a color definition for example, already causes a problem in the display.

For example, the definitions below in a new component in QT Creator give me the mirrored result, but in the slide component for squids there is a white background.

```
    id: presentation
    visible: true
    width: 440
    height: 680
    color: "transparent"

```
**Expected behavior**
Set a background color in the slide component and take advantage of the png image.

**Screenshots and Logs**
![image](https://user-images.githubusercontent.com/26740246/110219501-56bb9a80-7e9e-11eb-990d-f419e9f06e59.png)

**Comment by adriaandegroot** [comment]
See similar issue https://github.com/calamares/calamares-extensions/issues/9 ; this explains how and why the "basic" slideshow that ships with Calamares doesn't do backgrounds the way you expect.

And the example slideshow https://github.com/calamares/calamares-extensions/tree/calamares/branding/image-slideshow , which shows a slightly different approach to the whole thing. It comes down to this: the sample slideshow is probably over-engineered for use as a Calamares slideshow, and often the easiest way to fix it is to write other, simpler QML instead.

**Comment by hiukky** [comment]
@adriaandegroot  thanks for the quick reply and for the suggestions. I am following the tips and looking calmly at the other speeches and examples cited.

**Comment by hiukky** [comment]
@adriaandegroot  I followed what was suggested at https://github.com/calamares/calamares-extensions/issues/9#issuecomment-780926627 this example and it worked well!

I tried to use png but I had some problems with resizing, but with svg it was great.

Maybe in the near future I will try my own solution, maybe using SwipeView, but I still need to get some QT lessons lol.

Thank you and congratulations for the excellent work.
